client/extensions/woocommerce/app/settings/shipping/save-button.js

```diff
@@ -42,7 +45,22 @@ class ShippingSettingsSaveButton extends Component {
 	}
 
 	save = () => {
-		return null;
+		if ( ! config.isEnabled( 'woocommerce/extension-wcservices' ) ) {
+			return;
+		}
+
+		const { translate } = this.props;
+
+		const successAction = successNotice(
+			translate( 'Shipping settings saved' ),
+			{ duration: 4000, displayOnNextPage: true }
```

Comment:
The `displayOnNextPage` isn't needed, since this success action won't trigger a redirect.